README.md

```diff
@@ -1,5 +1,6 @@
 # Pantheon Ethereum Client
  [![Build Status](https://jenkins.pegasys.tech/job/Pantheon/job/master/badge/icon)](https://jenkins.pegasys.tech/job/Pantheon/job/master/)
+ [![Documentation Status](https://readthedocs.org/projects/pantheon/badge/?version=latest)](https://docs.pantheon.pegasys.tech/en/latest/?badge=latest)
```

Comment:
Second half of this link is where you go to when you click the badge, should be https://docs.pantheon.pegasys.tech/en/latest/ without the badge bit.

Reply:
Nevermind, it's correct, I should sleep rather than reviewing code.